rtdb2/RtDB2Monitor.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _INCLUDED_RTDB2MONITOR_H_
+#define _INCLUDED_RTDB2MONITOR_H_
+
+#include <string>
+#include <vector>
+
+class RtDB2Monitor
+{
+public:
+    ~RtDB2Monitor();
+    std::vector<std::string> getAgents();
```

Comment:
Is this returning agent paths or integers as string? Maybe document in this header with an example.

Reply:
pushed an update